### PR TITLE
✨ Combination of Simple mode pagination along with page size changer

### PR DESCRIPTION
**Pull request by Zian502**
fix https://github.com/ant-design/ant-design/issues/44290

### CODE REVIEW

**Comment by Gudivaka** [comment]
Thank you for the quick turnaround. when can we expect this to be merged and released?

File: tests/simple.test.js [hunk 1]
```diff
@@ -84,4 +85,40 @@ describe('simple Pagination', () => {
...
+    wrapper.find(Select).find('input').simulate('mousedown');
+    expect(wrapper.find(Select).find('.rc-select-item').at(2).text()).toBe(
+      '50 条/页',
+    );
+    const pageSize1 = wrapper.find(Select).find('.rc-select-item').at(0);
+    pageSize1.simulate('click');
+    expect(onChange).toBeCalled();
+    expect(onChange).toHaveBeenLastCalledWith(1, 10);
+  });
+
```
Comment (afc163) [inline tests/simple.test.js:123]:
```suggestion
```

**Comment by MadCcc** [comment]
test failed.

File: docs/examples/simple.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Pagination from 'rc-pagination';
+import Select from 'rc-select';
```
Comment (yoyo837) [inline docs/examples/simple.tsx:3]: 依赖需要从dev里移过来

**Comment by Gudivaka** [comment]
Hi @Zian502  Any update on this please. It would be great help if we get this out soon. Thank you.

File: docs/examples/simple.tsx (hunk 2 above)
Comment (MadCcc) [reply docs/examples/simple.tsx:3]: 应该不用吧，这里是 example

**Review by MadCcc — APPROVED** [review]

File: docs/examples/simple.tsx (hunk 2 above)
Comment (yoyo837) [reply docs/examples/simple.tsx:3]: > 应该不用吧，这里是 example

眼睛不好使了.

**Comment by Gudivaka** [comment]
Hi Team, when can we expect this changes to be released?

**Comment by Gudivaka** [comment]
Hi @MadCcc / @Zian502 , When can we expect this for the release.?

**Pull request merged**